example/image-classification/common/fit.py

```diff
@@ -168,10 +168,16 @@ def fit(args, network, data_loader, **kwargs):
 
     # callbacks that run after each batch
     batch_end_callbacks = [mx.callback.Speedometer(args.batch_size, args.disp_batches)]
+    eval_end_callbacks = []
     if 'batch_end_callback' in kwargs:
         cbs = kwargs['batch_end_callback']
         batch_end_callbacks += cbs if isinstance(cbs, list) else [cbs]
```

Comment:
@zihaolucky 
I assigned `eval_end_callbacks` with same value as `batch_end_ballbacks`, but log about validation set still print for each epoch, not `args.disp_batches`.
```Python
     batch_end_callbacks = [mx.callback.Speedometer(args.batch_size, args.disp_batches)]
    eval_end_callbacks = [mx.callback.Speedometer(args.batch_size, args.disp_batches)]
```